src/reference/antora/modules/ROOT/pages/mail.adoc

```diff
@@ -244,24 +253,138 @@ To use it, configure the following schema locations:
     https://www.springframework.org/schema/integration/mail/spring-integration-mail.xsd">
 ----
 
+[[configuring-outbound-channel-adapters]]
+== Configuring Outbound Channel Adapters
+
 To configure an outbound channel adapter, provide the channel from which to receive and the MailSender, as the following example shows:
 
-[source,xml]
+[tabs]
+======
+Java::
++
+[source,java,role="primary"]
+----
+@Bean
+@ServiceActivator(inputChannel = "outboundMail")
+public MessageHandler outboundMailMessageHandler(JavaMailSender mailSender) {
+    MailSendingMessageHandler handler = new MailSendingMessageHandler(mailSender);
+    return handler;
+}
+----
+Java DSL::
++
+[source,java,role="secondary"]
+----
+@Bean
+public IntegrationFlow mailOutboundFlow(JavaMailSender mailSender) {
+    return IntegrationFlow.from("outboundMail")
+            .handle(Mail.outboundAdapter(mailSender))
+            .get();
+}
+----
+Kotlin DSL::
++
+[source,kotlin,role="secondary"]
+----
+@Bean
+fun mailOutboundFlow(mailSender: JavaMailSender): IntegrationFlow {
+    return IntegrationFlow.from("outboundMail")
+        .handle(Mail.outboundAdapter(mailSender))
+        .get()
```

Comment:
This is not Kotlin idiomatic style.
Let me know if you'd like to pair on these samples!
Neither Groovy bellow.